### PR TITLE
added azetidine.yml

### PR DESCRIPTION
**Pull request by bchangala**
new yaml for azetidine -- hackathon

### CODE REVIEW

File: data/azetidine.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+chi_aa: 1.9260
+chi_bb: 2.6927
+chi_cc: -4.6187
+u_a: 0.53
+u_c: 1.23
+rep: "Ir"
+name: "azetidine"
+formula: "C3H7N"
+smiles: "N1CCC1"
+doi: "[10.1063/1.1334876, 10.1016/0022-2852(84)90251-0]"
```
Comment (laserkelvin) [inline data/azetidine.yml]: Not too big of a deal, but probably want the `doi` key to be interpreted as a list; remove the quotations

File: data/azetidine.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+A: 11453.13707
+B: 11339.19585
+C:  6612.03793
+Delta_J: 4.7388e-3
+Delta_JK: -5.0507e-3
+Delta_K: 7.6782e-3
+del_J: 1.36518e-3
+del_K: 0.96037e-3
+Phi_JK: -0.03653e-6
+Phi_K: 0.0365e-6
```
Comment (laserkelvin) [inline data/azetidine.yml:11]: Missing little phi_K? Should be 0.00749e-6

**Review by laserkelvin — CHANGES_REQUESTED** [review]
Just two lil things

**Review by tjbarnum13 — COMMENTED** [review]
all set

**Pull request merged**